lib/api-helper/listener/executeAutofixes.ts

```diff
@@ -105,17 +106,24 @@ export function executeAutofixes(registrations: AutofixRegistration[]): ExecuteG
             if (editResult.edited) {
                 // Send back an error code, because we want to stop execution of goals after this
```

Comment:
This comment seems to no longer be accurate.

Reply:
good point. update coming.